### ISSUE TITLE
Include sample-file for vcf stats step

### ISSUE THREAD

**Issue by srynobio** [frameshiftgenomics/mosaic-cli-releases#1]
`mosaic vcf [options] <VCF>` includes an option for specific samples collection (`-s, --sample`).  It would be nice if the bcftools option `sample-file` could be included to allow multiple samples.

From bcftools:

```
Options:
    -s, --samples <list>               list of samples for sample stats, "-" to include all samples
    -S, --samples-file <file>          file of samples to include
```

**Comment by chmille4** [comment]
Yep, that makes sense, I just added it to our issues board. Thanks for the idea!

**Comment by srynobio** [comment]
Also, It might be nice to allow a CPU option to pass through.  I've noticed that bcftools only run on a single cpu.  Could be sped up by allowing this.

**Comment by AlistairNWard** [comment]
@DoubleB123 If we're looking for more backend issues, this looks like a quick one.

**Comment by BrianBrenner** [comment]
@srynobio this is resolved in the latest release, v0.8.7. Let me know if you find any issues with it
